services/base/namespace.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: config-services-flux
\ No newline at end of file
+  name: flux-config-services
```

Comment:
This can be removed. we don't need this namespace.

Reply:
Ajay removed that in his changes.